internal/http/routes/api/v1/bookmarks.go

```diff
@@ -57,6 +58,63 @@ func (p *updateCachePayload) IsValid() error {
 	return nil
 }
 
+type contentResponseMessage struct {
```

Comment:
```suggestion
type readableResponseMessage struct {
```

Reply:
Can you move this struct closer to the handler that uses it? (right on top of it)